VirtoCommerce.Storefront.Model/Customer/Contracts/UpdateOrganizationResponseDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Storefront.Model.Customer.Contracts
+{
+    public class UpdateOrganizationResponseDto
```

Comment:
Do we actually need classes for each method response?

Reply:
I suppose , yes. look at ExperienceCartBuilder